### PR TITLE
refactor(loading-animation): Update CSS styles to match React implementation

### PR DESCRIPTION
**Pull request by jamesfan**
This PR updates the Loading Animation CSS styles so they match the React component. 

|                  | Visual | Code |
|------------------|--------|------|
| Breaking Changes | Yes    | Yes   |

### Visual
- Previously, the css was handling the positioning of the loading dots; now, the user will need to position the loading dots container themselves.

### Code
- `.wdc-loading-spinner` has been renamed to `.wdc-loading-dots`.

### Squash and Merge Message Proposal
refactor(loading-animation): update CSS styles to match React implementation (#83)

### CODE REVIEW

File: modules/loading-animation/css/lib/loading-dots.scss [hunk 1]
```diff
@@ -18,35 +18,36 @@
...
   width: $wdc-spacing-s;
   height: $wdc-spacing-s;
   display: block;
   position: absolute;
   font-size: 0px;
   content: ' ';
-  top: 50%;
-  left: 50%;
+  // top: 50%;
+  // left: 50%;
```
Comment (mannycarrera4) [inline modules/loading-animation/css/lib/loading-dots.scss]: do we still want these commented out?

Comment (jamesfan) [reply modules/loading-animation/css/lib/loading-dots.scss]: Whoops, no we don't :) Careless error, I'll remove the commented-out styles.

Comment (mannycarrera4) [reply modules/loading-animation/css/lib/loading-dots.scss]: all good! Just checking :)

Comment (jamesfan) [reply modules/loading-animation/css/lib/loading-dots.scss]: Thanks for the catch!

**Comment by jamesfan** [comment]
@anicholls I've pushed out a fix that styles `.wdc-loading-dots` as a container using your suggested styles. Let me know if you see any other issues.

**Pull request merged**